template/composer.json

```diff
@@ -15,7 +15,7 @@
         "drupal/cog": "^1.0.0",
         "drupal/core": "^8.0",
         "drupal/features": "^3.0.0",
-        "drupal/memcache": "^2.0.0-alpha2",
+        "drupal/memcache": "2.x-dev",
```

Comment:
Hmm I don't think there's a patch against it but I can still lock it to a hash tomorrow.